### PR TITLE
L breezy branch

### PR DESCRIPTION
**Pull request by luciebrock**
This doesn't have errors that I'm aware of.
I will remove payment from the navbar when I am done using it, I just want it there so I can check in on the payment page.

### CODE REVIEW

File: app/controllers/sessions_controller.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 class SessionsController < ApplicationController
 
-  skip_before_action :authorized, only: [:new, :create, :welcome, :about, :map, :faq, :pricing] #anything listed here can be viewed w/o logging in/authorization
+  skip_before_action :authorized, only: [:new, :create, :welcome, :about, :map, :faq, :pricing, :payment] #anything listed here can be viewed w/o logging in/authorization
```
Comment (epartakki) [inline app/controllers/sessions_controller.rb:3]: great job @sunniraleigh

File: app/views/sessions/about.html.erb [hunk 2]
```diff
@@ -1,12 +1,18 @@
 <h1>About Us</h1>
 <section>
-    <p>Founded in 2022 by the Kales Collective, ValetBike is Nipmuc Notch's premiere bike share co-op app. Since our inception, we've been dedicated to providing a new equitable method of transportation for all our users. ValetBike has an affordable, environmentally friendly option for everyone, whether you're looking for a leisure ride on the weekend or a way to work.</p>
+    <p>KALES is a small team of undergraduate students, founded in January 2022. After seeing dire need for a local bike sharing app in Nipmuc Notch, our team set out with the goal of providing a new equitable method of transportation for all our users. ValetBike has an affordable, environmentally friendly option for everyone, whether you're looking for a leisure ride on the weekend or a way to work. Outside of software engineering, our team likes to play minecraft, chess, and complain. </p>
```
Comment (kvangreen) [inline app/views/sessions/about.html.erb:3]: Delete

Comment (epartakki) [inline app/views/sessions/about.html.erb:3]: Thank you!

File: app/views/sessions/about.html.erb [hunk 3]
```diff
@@ -1,12 +1,18 @@
...
 <section>
 <h2>Meet Our Team</h2>
-    <p>LUCIE BROCK (they/them) is such a welcoming person. I want to do the same for them.</p>
-    <p>ALINE MARRA (she/her) is one of my best friends. Genuinely, let me give her the world.</p>
-    <p>ELENI PARTAKKI (she/her) intimidates me but in the way that makes me want to be the best version of myself.</p>
-    <p>SUNNI RALEIGH (she/they) has the best vibes ever, and it's kind of homophobic that we only became friends this semester.</p>
-    <p>KAT VAN GREEN (she/they) is a front-end developer and aspiring archivist. Outside of ValetBike, they're currently working on a story-based game about lighthouses, ghost, and memory.</p>
+    <p>LUCIE BROCK (they/them)</p>
+        <p>chair of the vibes committee</p>
+    <p>ALINE MARRA (she/they)</p>
```
Comment (kvangreen) [inline app/views/sessions/about.html.erb:10]: she/her

**Review by sunniraleigh — APPROVED** [review]
looks good

File: app/views/sessions/about.html.erb [hunk 4]
```diff
@@ -1,12 +1,18 @@
...
+	    <p>lead critic</p>
+    <p>ELENI PARTAKKI (she/her)</p>
+	    <p>squire of positivity</p>
```
Comment (epartakki) [inline app/views/sessions/about.html.erb:13]: what is squire.

File: app/views/sessions/about.html.erb [hunk 5]
```diff
@@ -1,12 +1,18 @@
...
 </section>
+
 <section>
 <h2>Meet Our Team</h2>
-    <p>LUCIE BROCK (they/them) is such a welcoming person. I want to do the same for them.</p>
-    <p>ALINE MARRA (she/her) is one of my best friends. Genuinely, let me give her the world.</p>
-    <p>ELENI PARTAKKI (she/her) intimidates me but in the way that makes me want to be the best version of myself.</p>
-    <p>SUNNI RALEIGH (she/they) has the best vibes ever, and it's kind of homophobic that we only became friends this semester.</p>
-    <p>KAT VAN GREEN (she/they) is a front-end developer and aspiring archivist. Outside of ValetBike, they're currently working on a story-based game about lighthouses, ghost, and memory.</p>
+    <p>LUCIE BROCK (they/them)</p>
```
Comment (kvangreen) [inline app/views/sessions/about.html.erb:8]: NAUR let us not use our full names

**Review by epartakki — APPROVED** [review]
Great work LB! Keep us posted on changes.

**Review by kvangreen — CHANGES_REQUESTED** [review]
Looks great babe just let us get rid of our full names ASAP

**Review by kvangreen — APPROVED** [review]

**Review by amarrap — APPROVED** [review]
looks good :)

**Pull request merged**